codersdk/users.go

```diff
@@ -93,6 +93,12 @@ type UserRoles struct {
 	OrganizationRoles map[uuid.UUID][]string `json:"organization_roles"`
 }
 
+type ConvertLoginRequest struct {
+	// ToLoginType is the login type to convert to.
+	ToLoginType LoginType `json:"to_login_type" validate:"required"`
+	LoginWithPasswordRequest
```

Comment:
It's weird to embed this when the `email` isn't valuable here. Can we just have `Password` instead?

Reply:
Removed